docs/FeatureManagement/Samples/NoFilters.tests.json

```diff
@@ -0,0 +1,90 @@
+[
+    {
+        "FeatureFlagName": "BooleanTrue",
+        "Inputs": {},
```

Comment:
nit: We could omit `Input` when it's empty just to reduce the size of this even more. But I do see some value in being explicit.

Reply:
I expect it will be required on most of them once we add variants.